.gitignore

```diff
@@ -1,3 +1,7 @@
+
+My settings for Mac and Idea
```

Comment:
Please remove or comment it (with #). As if you leave it as is it would search "My settings for Mac and Idea" for ignoring